sdk/src/transaction/error.rs

```diff
@@ -135,6 +135,10 @@ pub enum TransactionError {
     /// Transaction would exceed max Vote Cost Limit
     #[error("Transaction would exceed max Vote Cost Limit")]
     WouldExceedMaxVoteCostLimit,
+
+    /// Transaction would exceed max total account data limit
+    #[error("Transaction would exceed max total account data limit")]
+    WouldExceedMaxTotalAccountDataLimit,
```

Comment:
Seems like there's already a `WouldExceedMaxAccountDataCostLimit`?

Reply:
Yep!
https://github.com/solana-labs/solana/blob/6c56eb9663ef0280cf7f53202425d6530d9e6ca4/sdk/src/transaction/error.rs#L105-L107

Since that error specifically says _within a block_, I figured that a new error for exceeding the total accounts data size would be useful. I can reuse the existing error too; whichever is preferred.

Reply:
I think reusing this makes sense to me, falls under the same category

Reply:
convo moved here: https://github.com/solana-labs/solana/pull/23422#discussion_r818138044